### PR TITLE
Adds new Title Screens from Contest 2024

### PR DESCRIPTION
**Pull request by JamieD1**
Adds new Title Screens from Contest

Changed the way it picks the screens because it was complicated and dumb,
Simple easy way now, Normal/Joke/Rare have own folders, put in folder, job done.

Weights are defined as:
5% Rare,
20% Joke
74% Normal/Serious

DO NOT MERGE UNTIL I SAY WE CAN, AWAITING NEW IMAGES BECAUSE FILE SIZES ARE HOOOOOOOOOOOOOOOGE

:cl:  
rscadd: Added new Title Screens from 2024 Contest
/:cl:

### CODE REVIEW

**Comment by neeshacark** [comment]
should add the joke ones to the april fools list so they always roll on that day

**Comment by JamieD1** [comment]
> should add the joke ones to the april fools list so they always roll on that day

Good idea

**Review by ToasterBiome — APPROVED** [review]

File: code/controllers/subsystem/title.dm [hunk 1]
```diff
@@ -19,17 +19,33 @@ SUBSYSTEM_DEF(title)
...
 	fdel("data/previous_title.dat")
 
-	var/list/provisional_title_screens = flist("[global.config.directory]/title_screens/images/")
+	var/list/normal_provisional_title_screens = flist("[global.config.directory]/title_screens/images/normal/")
+	var/list/joke_provisional_title_screens = flist("[global.config.directory]/title_screens/images/joke/")
+	var/list/rare_provisional_title_screens = flist("[global.config.directory]/title_screens/images/rare/")
 	var/list/title_screens = list()
-	var/use_rare_screens = prob(1)
+	var/use_rare_screens = prob(1)		// 1% Chance for Rare Screens in /rare
+	var/use_joke_screens = prob(20) 	// 10% Chance for Joke Screens in /joke
```
Comment (Bibby0110) [inline code/controllers/subsystem/title.dm]: you do numbers good?

File: code/controllers/subsystem/title.dm [hunk 2]
```diff
@@ -19,17 +19,33 @@ SUBSYSTEM_DEF(title)
...
 
-	for(var/S in provisional_title_screens)
-		var/list/L = splittext(S,"+")
-		if((L.len == 1 && (L[1] != "exclude" && L[1] != "blank.png"))|| (L.len > 1 && ((use_rare_screens && lowertext(L[1]) == "rare") || (lowertext(L[1]) == lowertext(SSmapping.config.map_name)))))
+	if(SSevents.holidays && SSevents.holidays[APRIL_FOOLS])
+		use_joke_screens = 1
```
Comment (Bibby0110) [inline code/controllers/subsystem/title.dm]:
```suggestion
		use_joke_screens = TRUE
```

File: code/controllers/subsystem/title.dm (hunk 1 above)
Comment (JamieD1) [inline code/controllers/subsystem/title.dm]:
```suggestion
	var/use_joke_screens = prob(20) 	// 20% Chance for Joke Screens in /joke
```

**Pull request merged**